<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoClass</class>
 <widget class="QMainWindow" name="DemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>128</width>
          <height>225</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QListWidget学习</string>
        </attribute>
        <attribute name="toolTip">
         <string>helloworld</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QToolButton" name="toolButton_5">
             <property name="text">
              <string>init</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_6">
             <property name="text">
              <string>clear</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_7">
             <property name="text">
              <string>insert</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_8">
             <property name="text">
              <string>add</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_17">
             <property name="text">
              <string>delete</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>28</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTreeWidget学习</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>28</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTableWidget学习</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>list widget</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>当前项变化</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>CheckBox</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QToolButton" name="toolButton">
                <property name="text">
                 <string>item</string>
                </property>
                <property name="popupMode">
                 <enum>QToolButton::MenuButtonPopup</enum>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_2">
                <property name="text">
                 <string>inv</string>
                </property>
                <property name="popupMode">
                 <enum>QToolButton::DelayedPopup</enum>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_3">
                <property name="text">
                 <string>all not</string>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_4">
                <property name="text">
                 <string>all</string>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>tree widget</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>table widget</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInit"/>
   <addaction name="actionClear"/>
   <addaction name="actionInsert"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionInit">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>进行初始化</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>进行清除</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>进行插入</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>进行添加</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>进行删除</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>进行全选</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>进行全不选</string>
   </property>
  </action>
  <action name="actionInverse">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>进行反选</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>进行退出</string>
   </property>
  </action>
  <action name="actionItem">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>进行项选择</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>DemoClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
